### ISSUE TITLE
Document the myStyle property

### ISSUE THREAD

**Issue by gaurav21r** [FremyCompany/css-grid-polyfill#25]
I was working with jQuey using ````$('#myelem').css('grid-row', '1/12')```` when I noticed the error message saying that ````HTMLDOMElement.myStyle```` is to be modified and not ````HTMLDOMElement.style```` (which jQuery and other frameworks change).

I think this should be properly documented. Sorry if I missed it somewhere!

**Comment by FremyCompany** [comment]
Yeah, that's a ugly limitation due to Chrome and Safari inability to properly redefine properties on DOM elements. Chrome fixed the bug recently, but I don't want to change the way it works knowing it will fail in older browsers on which developers will not test on. 

In the mean time, I would advise only using classes to change the style, and not change the inline style. There are likely bugs related to this, I'm not sure it works in all cases anyway.

You're right I should document this more prominently.
